src/main/java/de/tum/in/www1/artemis/service/ProgrammingExerciseExportService.java

```diff
@@ -277,15 +278,15 @@ private void createZipFile(Path zipFilePath, List<Path> paths) throws IOExceptio
     }
 
     /**
-     * Adds the student id of the given student participation to the project name in all .project (Eclipse)
+     * Adds the student/team identifier of the given student participation to the project name in all .project (Eclipse)
      * and pom.xml (Maven) files found in the given repository.
      *
      * @param repo The repository for which the student id should get added
      * @param programmingExercise The checked out exercise in the repository
-     * @param participation The student participation for the student id, which should be added.
+     * @param participation The student participation for the student/team identifier, which should be added.
      */
     public void addStudentIdToProjectName(Repository repo, ProgrammingExercise programmingExercise, StudentParticipation participation) {
-        String studentId = participation.getStudent().getLogin();
+        String studentId = participation.getParticipantIdentifier();
```

Comment:
I guess the variable `studentId`, should then be renamed to `participantId`

Reply:
Renamed it to `participantIdentifier`.